jax/_src/scipy/special.py

```diff
@@ -1048,10 +1051,17 @@ def _create_polynomial(var, coeffs):
                 jnp.where(z >= dtype(8.0), x_for_small_p, x_otherwise))
 
   x = jnp.where(p > dtype(1. - np.exp(-2.)), x, -x)
-  infinity = jnp.full(shape, dtype(np.inf))
-  x_fix_boundaries = jnp.where(
-      p == dtype(0.0), -infinity, jnp.where(p == dtype(1.0), infinity, x))
-  return x_fix_boundaries
+  with config.debug_infs(False):
+    infinity = jnp.full(shape, dtype(np.inf))
+    x = jnp.where(
+        p == dtype(0.0), -infinity, jnp.where(p == dtype(1.0), infinity, x))
+  if not isinstance(x, core.Tracer):
+    try:
+      dispatch.check_special("ndtri", [x])
```

Comment:
What does `check_special` do? I haven't seen it before, and the function doesn't have any documentation or comments.

Reply:
Oh, looks like it just checks for nans and infs...